src/components/NotePopup/NotePopup.js

```diff
@@ -79,7 +74,7 @@ function NotePopup(props) {
         className={notePopupButtonClass}
         onClick={togglePopup}
         onKeyDown={e => {
-          if (e.key === 'Enter' || e.key === ' ') {
+          if (e.key === 'Enter') {
```

Comment:
Add comment here that space was removed because of this. Just so we don't get confused if we get that bug in the future.

Reply:
The space there is not a problem actually. I can return this line